lib/srv/reexec.go

```diff
@@ -165,10 +165,7 @@ func RunCommand() (io.Writer, int, error) {
 		errorWriter = tty
 		err = uacc.Open(c.UaccMetadata.UtmpPath, c.UaccMetadata.WtmpPath, c.Login, c.UaccMetadata.Hostname, c.UaccMetadata.RemoteAddr, tty)
 		if err != nil {
-			// the error is critical and we should fail command execution
-			if !trace.IsAccessDenied(err) && !trace.IsNotFound(err) {
-				return errorWriter, teleport.RemoteCommandFailure, trace.Wrap(err)
-			}
+			// uacc support is best-effort, just disable it and continue
```

Comment:
nit: Move it to log debug ?

Reply:
I considered that, but at this point teleport has re-executed itself and stdout and stderr will be used for the output of the child command and returned to the user rather than written to the teleport log